json_minify/test_json_minify.py

```diff
@@ -53,6 +73,19 @@ class JsonMinifyTestCase(unittest.TestCase):
 
             }
             ''',
+            '''
+             \x20
```

Comment:
I don't see the issue here. `textwrap.dedent` is never called on the second argument of the test fixtures...